web/app.js

```diff
@@ -41,6 +41,12 @@ import { SecondaryToolbar } from 'pdfjs-web/secondary_toolbar';
 import { Toolbar } from 'pdfjs-web/toolbar';
 import { ViewHistory } from 'pdfjs-web/view_history';
 
+var {
```

Comment:
Please note that https://github.com/mozilla/pdf.js/blob/master/web/debugger.js is currently being initialized with the *entire* `pdfjsLib` object, and I just didn't fancy re-factoring that code *too* in this already quite large PR.